contracts/dao/neutron-chain-manager/src/testing/mock_querier.rs

```diff
@@ -43,37 +55,69 @@ impl WasmMockQuerier {
     pub fn handle_query(&self, request: &QueryRequest<Empty>) -> QuerierResult {
         match &request {
             #[allow(deprecated)]
-            QueryRequest::Stargate { path, data: _ } => match path.as_str() {
+            QueryRequest::Grpc(GrpcQuery { data: _, path }) => match path.as_str() {
                 "/neutron.cron.Query/Params" => {
```

Comment:
is it possible to reuse some constants here instead of these raw strings like you did in `contract.rs` `check_proposal_execute_message` `cron::QueryParamsRequest::TYPE_URL`?